### PR TITLE
workflow engine operator proxy

### PR DESCRIPTION
**Pull request by vrancurel**
# Pull request template

## Description
Proxy for Orbit to the workflow-engine-operator
### Motivation and context
Experimental workflow engine feature
Why is this change required? What problem does it solve?
Use a proxy to access internal operator

### CODE REVIEW

**Review by rachedbenmustapha — COMMENTED** [review]
Since the requests are not actually going to backbeat but to w-e-o why not move it out of the backbeat namespace?

When troubleshooting It's important to know that all of /_/backbeat goes to backbeat, all of /_/workflow-engine-operator goes to w-e-o etc

**Review by dora-korpar — APPROVED** [review]

**Comment by vrancurel** [comment]
@bert-e reset

File: lib/routes/routeWorkflowEngineOperator.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+function routeWorkflowEngineOperator(clientIP, request, response, log) {
+    log.debug('routing request', {
+        method: 'routeWorkflowEngineOperator',
+        url: request.url,
+    });
+    _normalizeRequest(request);
+    const requestContexts = prepareRequestContexts('objectReplicate', request);
+
+    // proxy api requests to Backbeat API server
```
Comment (rahulreddy) [inline lib/routes/routeWorkflowEngineOperator.js]: proxy api requests to ~Backbeat~ WorkflowEngine API server

**Review by rahulreddy — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Review by bennettbuchanan — APPROVED** [review]

**Comment by vrancurel** [comment]
@bert-e approve

**Pull request merged**